Cargo.toml

```
@@ -149,6 +149,7 @@ wasmcloud-provider-sqldb-postgres = { workspace = true, optional = true }
 wasmcloud-tracing = { workspace = true, features = ["otel"], optional = true }
 
 [dev-dependencies]
+axum ={ workspace = true, features = ["query"]}
```

```suggestion
axum = { workspace = true, features = ["query"] }
```

The superest of super nits, since this will trigger a format later 😅